content/en/Community/_index.md

```diff
@@ -186,7 +186,21 @@ The following responsibilities apply to the subproject for which one would be an
 
 **Off-boarding Criteria**
 
-- Voted out by a majority of current maintainers via a GitHub issue during a voting period lasting seven (7) days. A vote may be called by any maintainer after the point at which the responsibilities have not been met. A positive vote will result in movement to an _emeritus_ status within `MAINTAINERS.md` and removal from organization membership.
+An off-boarding vote may be called by any maintainer if any of the following criteria are met:
+- A maintainer has made less than 30 contributions over a span of 6 months.
```

Comment:
As I couldn't join the meeting: How was the number `30` decided?

Reply:
I probably should have waited a little longer before merging this. Those who attended agreed on the number because the way those devstats are calculated they pick up every tiny change made to a repo anywhere, for example editing a comment. We need some data-driven, evidence-backed threshold that can be used to qualify when a vote _can_ be called (doesn't mean it will). Having a number like 30 out of six months we thought was reasonable.

Reply:
Per the devstat [here](https://kyverno.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%206%20months&var-metric=contributions&var-repogroup_name=All&var-country_name=All&from=1522810884223&to=1680577284223&var-companies=All), 30 makes the contributor in ~top 20 which looks reasonable to me.

Reply:
Thanks for the explanations - sounds reasonable to me :+1: